maven-release-manager/src/main/mdo/release-descriptor.mdo

```diff
@@ -491,10 +491,10 @@
         <field>
           <name>projectVersionPolicyConfig</name>
           <version>3.0.0+</version>
-          <type>String</type>
+          <type>DOM</type>
```

Comment:
DOM is exactly what you need. Just like here: https://github.com/apache/maven/blob/8125b3131f2e0f88837040b0baf4e0aaa8210679/maven-settings/src/main/mdo/settings.mdo#L574-L582 

 I assumed that I have provided this pointer to you.